### PR TITLE
Add getting started guide

### PR DESCRIPTION
**Pull request by MarcelMue**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,48 @@
...
+- `/test aws` runs `awscnfm` against the release - this only works for AWS.
+
+There are multiple dashboards which can be useful to track the progress of tests:
+- [prow](https://prow.rfjh2.k8s.gorilla.eu-central-1.aws.gigantic.io/) gives a rough overview of the running tests.
+- [tekton](https://tekton.rfjh2.k8s.gorilla.eu-central-1.aws.gigantic.io/#/pipelineruns) gives a more in depth overview of the test progress,
+- [kibana](https://kibana.rfjh2.k8s.gorilla.eu-central-1.aws.gigantic.io/app/kibana#/dashboard/a0274d90-151c-11eb-a856-f95f4e179788?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:now-15m,to:now))&_a=(description:'',filters:!(('$state':(store:appState),meta:(alias:!n,controlledBy:'1603451084569',disabled:!f,index:'83f5e030-146a-11eb-a856-f95f4e179788',key:kubernetes.labels.prow_k8s_io%2Frefs_repo.keyword,negate:!f,params:(query:releases),type:phrase),query:(match_phrase:(kubernetes.labels.prow_k8s_io%2Frefs_repo.keyword:releases)))),fullScreenMode:!f,options:(hidePanelTitles:!f,useMargins:!t),query:(language:kuery,query:''),timeRestore:!f,title:'Prow%20Dashboard',viewMode:view)) gives full insight of the logs emitted during your test.
```
Comment (stone-z) [inline README.md:19]: That URL though

File: README.md [hunk 2]
```diff
@@ -2,6 +2,48 @@
...
+
+You can find guides for the creation of new tests in the `docs` subfolder.
+
+Deploying `test-infra` is currently a bit cumbersome as it is running as an app inside a tenant cluster.
+The tenant cluster is `rfjh2` on `gorilla`.
+To update `test-infra` you will need to manually edit the `app` CR on the `gorilla` CP in the namespace `rfjh2`.
+**There is currently no automatic rollout by merging to master!**
+
+The `roles` and `serviceaccounts` on the target CPs are currently created manually.
+You can find the resources in the [here](control-plane/README.md).
```
Comment (stone-z) [inline README.md:42]:
```suggestion
You can find the resources [here](control-plane/README.md).
```

**Review by stone-z — APPROVED** [review]
One small suggestion, otherwise this improves things

File: README.md (hunk 1 above)
Comment (MarcelMue) [reply README.md:19]: A real beauty :D

**Pull request merged**